app/adapters/application.js

```diff
@@ -1,29 +1,25 @@
 import JSONAPIAdapter from '@ember-data/adapter/json-api';
 const maxRetries = 5;
-export default JSONAPIAdapter.extend({
-  init() {
-    this._super(...arguments);
-  },
+export default class ApplicationAdapter extends JSONAPIAdapter {
   ajax(url, method) {
     if (method === 'POST')
-      return this._super(...arguments);
+      return super.ajax(...arguments);
 
-    const origAjax = this._super;
     const args = arguments;
     const sleep = (time) => new Promise( (resolve/*,reject*/) => setTimeout(() => resolve(true), time));
     const retry = (livesSpent) => {
       return new Promise((resolve, reject) => {
         sleep(250 * livesSpent).then(() => {
           if (livesSpent < maxRetries )
-            origAjax.apply(this,args).then( (res) => resolve(res), () => retry(livesSpent + 1).then(
+            super.ajax(...args).then((res) => resolve(res), () => retry(livesSpent + 1).then(
```

Comment:
I need to double check this tomorrow.

Reply:
Fixed the issue and rewrote the method while I was at it. I think it's more readable now (and easier to extract into a shareable util).